src/sail_sv_backend/jib_sv.ml

```diff
@@ -2338,8 +2354,18 @@ module Make (Config : CONFIG) = struct
       defs = List.map mk_def defs;
     }
 
-  let rec pp_module m =
+  let rec pp_module ?(prepend_globals = true) m =
     let params = if m.recursive then space ^^ string "#(parameter RECURSION_DEPTH = 10)" ^^ space else empty in
+    let prepend_globals =
```

Comment:
Doesn't make sense to take `prepend_globals` as a parameter then immediately redefine it. I'm not sure I understand what this is trying to do anyway?